content/docs/gto/command-reference/annotate.md

```diff
@@ -0,0 +1,86 @@
+# annotate
+
+Update artifact metadata annotations.
```

Comment:
Can we define this without using the work annotation? Also, it's "Create or update" right? Maybe just "Write" ?

Reply:
"Create or update metainformation about an artifact"?